vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_havana_errors_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 describe EmsRefresh::Refreshers::OpenstackRefresher do
   before(:each) do
+    EmsRefresh.debug_failures = false
+
```

Comment:
This spec (note filename) is specifically intended to confirm that we do the right thing with `last_refresh_status` / `last_refresh_error`, and presumably, at least by implication, that we continue trying to process anything else.

Now, it could be observed that there are very few assertions going on, in these only specs that care about the "continue" behaviour... but the lack of coverage there is obviously not new in this commit.